library/spdm_requester_lib/libspdm_req_get_event_types.c

```diff
@@ -94,10 +94,6 @@ static libspdm_return_t libspdm_try_get_event_types(libspdm_context_t *spdm_cont
     }
 
     /* -=[Validate Response Phase]=- */
-    if (spdm_response_size < sizeof(spdm_message_header_t)) {
```

Comment:
I think this step is still needed, and should be changed to `if (spdm_response_size < sizeof(spdm_supported_event_types_response_t))`

The reason is that, the following code will access the fields of the message. This step is to guarantee those fields are valid in response buffer.

Reply:
Ok I've re-instated the check, but using `sizeof(spdm_supported_event_types_response_t)`.

Reply:
Ah, agree. For error.